sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/ModelHelper.java

```diff
@@ -346,4 +348,33 @@ public static PageBlobCopyIncrementalRequestConditions populateBlobDestinationRe
             .setIfUnmodifiedSince(requestConditions.getIfUnmodifiedSince())
             .setTagsConditions(null);
     }
+
+    public static String getObjectReplicationDestinationPolicyId(Map<String, String> objectReplicationStatus) {
+        objectReplicationStatus = objectReplicationStatus == null ? new HashMap<>() : objectReplicationStatus;
+        return objectReplicationStatus.getOrDefault("policy-id", null);
+    }
+
+    public static List<ObjectReplicationPolicy> getObjectReplicationSourcePolicies(
```

Comment:
This was refactored from somewhere else?

Reply:
yeah from BlobProperties